configure-pull-request/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Configure Pull Request
+
+description: Sets the parameters of an existing Pull Request at once
+
+inputs:
+  github-token:
+    description: A GitHub personal access token with write access to the project
+    required: true
+  labels:
+    description: Comma-separated list of labels
+    required: false
+  project-url:
+    description: URL of a GitHub project
+    required: false
+  reviewers:
+    description: Comma-separated list of users
+    required: false
+  team-reviewers:
+    description: Comma-separated list of teams
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Set labels
+      uses: buildsville/add-remove-label@eeae411a9be2e173f2420e1644514edbecc4e835 # pin@v2.0.0
+      with:
+        labels: ${{ inputs.labels }}
+        token: ${{ inputs.github-token }}
+        type: add
+      if: ${{ inputs.labels != '' }}
+    - name: Set project
+      uses: actions/add-to-project@960fbad431afda394cfcf8743445e741acd19e85 # pin@v0.4.0
+      with:
+        github-token: ${{ inputs.github-token }}
+        project-url: ${{ inputs.project-url }}
+      if: ${{ inputs.project-url != '' }}
+    - name: Set reviewers (users)
+      uses: dannysauer/actions-assigner@ca2ca780928294ee5126986cc31db6c2ee7d69b0 # pin@v2.0.1
+      with:
+        reviewers: ${{ inputs.reviewers }}
+        token: ${{ inputs.github-token }}
+      if: ${{ inputs.reviewers != '' }}
+    - name: Set reviewers (teams)
+      uses: dannysauer/actions-assigner@ca2ca780928294ee5126986cc31db6c2ee7d69b0 # pin@v2.0.1
```

Comment:
will Renovate update this action in the future if a new version is released or will it be pinned to 2.0.1 permanently?

Reply:
I am not familiar with Renovate configuration but it seems that digest can be updated. What is the Infra team strategy to deal with (small) external action ?

Reply:
We usually just specify the semver (i.e., `uses: dannysauer/actions-assigner@2.0.1`)

Renovate also supports hash pinning with following the updates: https://docs.renovatebot.com/modules/manager/github-actions/#additional-information

Reply:
> We usually just specify the semver (i.e., `uses: dannysauer/actions-assigner@2.0.1`) and Renovate creates PR when a new version is released.
> 
> Renovate also supports hash pinning with following the updates: https://docs.renovatebot.com/modules/manager/github-actions/#additional-information

I will specify the semver for now. Thx for the link.